compiler/rustc_resolve/src/lib.rs

```diff
@@ -1506,6 +1510,34 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
         self.tcx.untracked().cstore.leak();
     }
 
+    fn check_reexport_ambiguities(
```

Comment:
`ImportResolver` is removed now so this function can be moved into `imports.rs` next to `finalize_impors`.